package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "flask-apispec",
-  "version": "0.3.1",
+  "version": "0.6.0",
   "dependencies": {
-    "swagger-ui": "2.1.3"
+    "swagger-ui-dist": "3.9.3"
```

Comment:
Why switch the package? Can we stick to using the official `swagger-ui` package?

Reply:
Oh, I see now...

> swagger-ui-dist is a dependency-free module that includes everything you need to serve Swagger-UI in a server-side project, or a single-page application that can't resolve npm module dependencies.

https://www.npmjs.com/package/swagger-ui